src/services/transaction_watcher_signer_service.ts

```diff
@@ -345,9 +404,21 @@ export class TransactionWatcherSignerService {
                 }
                 if (tx.blockNumber + NUMBER_OF_BLOCKS_UNTIL_CONFIRMED < latestBlockNumber) {
                     tx.status = TransactionStates.Confirmed;
+                    this._transactionsUpdateCounter.inc({ signer_address: tx.from, status: tx.status }, 1);
                     await this._transactionRepository.save(tx);
                 }
             }
         }
     }
+    private async _updateSignerBalancesAsync(): Promise<void> {
+        for (const signerAddress of this._availableSignerPublicAddresses) {
+            // TODO(oskar) - use contract to grab all balances in a single RPC call?
+            const signerBalance = await this._web3Wrapper.getBalanceInWeiAsync(signerAddress);
+            this._signerBalancesGauge.set(
+                { signer_address: signerAddress },
+                // tslint:disable-next-line:custom-no-magic-numbers
+                Web3Wrapper.toUnitAmount(signerBalance, 18).toNumber(),
```

Comment:
we probbably have an `ETH_DECIMALS`  somewhere if you want to import this instead of magic number with tslint disable

Reply:
`grep`ing didn't help, settled for our own constants.